packages/init/test/test-async_hooks.js

```diff
@@ -2,7 +2,7 @@
 
 // Use a package self-reference to go through the "exports" resolution
 // eslint-disable-next-line import/no-extraneous-dependencies
-import '@endo/init';
+import '@endo/init/debug.js';
```

Comment:
Reviewers, since this is in `packages/init`, I'd like feedback on whether this change is appropriate before merging.

Reply:
I defer to @mhofman’s opinion on this test.

Reply:
If debug transitively imports pre throught the package.json module map, i think it's ok

Reply:
Thinking about it, it should since it's a public entry point itself.